test/test_regression.py

```diff
@@ -45,7 +45,8 @@
 download_md5 = [
     ('mt-st-helens-veg', 'd5782e07241cb3fe9f5b2e1bb804a794'),
     ('bird-size', '45c7507ae945868c71b5179f7682ea9c'),
-    ('mammal-masses', 'b54b80d0d1959bdea0bb8a59b70fa871')
+    ('mammal-masses', 'b54b80d0d1959bdea0bb8a59b70fa871'),
+    ('eltron-trails', '4813182220482d30210eb918d6decbee127032dc')
```

Comment:
since `mt-st-helens-veg` is a dataset with multi tables, lets remove elton-traits from the list of `download_md5`
and instead add it to `db_md5`.
Also use correct spelling.